config/calico_crds.yml

```diff
@@ -0,0 +1,6 @@
+# The Makefile goals for read-calico-crds-version and read-calico-enterprise-crds-version read from here what branch
+# on the calico repo to use for downloading the ./libcalico CRDs.
+calico:
```

Comment:
I think we can continue to use the current calico and enterprise versions files. I'd even suggest keeping the libcaclico-go name as I think changing the field will have other knock on effects to the build and release pipelines.